examples/multipleCalls/out/gas/Callee_tempDelta_pass.kast

```diff
@@ -0,0 +1 @@
+#if notBool ABI_x ==K 0 #then 20491 #else 5491 #fi
```

Comment:
plz don't include these

Reply:
They are needed since these rules are used as lemmas in other specs